neqo-transport/Cargo.toml

```diff
@@ -16,15 +16,15 @@ readme.workspace = true
 workspace = true
 
 [dependencies]
-# Sync with https://searchfox.org/mozilla-central/source/Cargo.lock 2024-02-08
-enum-map = { version = "2.7", default-features = false }
+# Checked against https://searchfox.org/mozilla-central/source/Cargo.lock 2024-11-11
+enum-map = { workspace = true }
 indexmap = { version = "2.2", default-features = false } # See https://github.com/mozilla/neqo/issues/1858
 log = { workspace = true }
 neqo-common = { path = "../neqo-common" }
 neqo-crypto = { path = "../neqo-crypto" }
 qlog = { workspace = true }
-smallvec = { version = "1.11", default-features = false }
-static_assertions = { version = "1.1", default-features = false }
+smallvec = { version = "1.13", default-features = false }
```

Comment:
Why this minor version bump? Is there a bug in `smallvec` `1.11` that is fixed in `1.13`. Unless there is a good reason for it, I don't think we should bump non-breaking versions, in this case the minor bump.

Don't feel strongly. In my eyes it adds noise. In addition, a consumer of `neqo-transport` might prefer using `neqo-transport` with `smallvec` `1.11`, e.g. as they have other dependencies using `smallvec` `1.11`.

Reply:
Gecko did the same bump